content/posts/paginas_acessiveis_hatemile.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Páginas acessíveis com o HaTeMiLe
+author: Carlson Santana Cruz
+type: post
+image: http://imgh.us/dog-41431_1280.png
+date: 2017-05-07
+excerpt: O HaTeMiLe é uma biblioteca que utiliza os padrões web para tentar resolver alguns problemas de acessibilidade, mas como utilizá-la e o que ela realmente faz?
+categories:
+  - HTML
+  - JavaScript
+  - Acessibilidade
+---
+
+A acessibilidade finalmente esta se tornando uma das metas no desenvolvimento web, mesmo assim ela enfrenta um conjunto de problemas:
+
+ 1. nas aulas, apostilas e tutoriais sobre desenvolvimento web dificilmente abordam a acessibilidade;
+ 2. mesmo conhecendo os padrões web (HTML, CSS e JavaScript) o desenvolvedor pode acabar desenvolvendo páginas web inacessíveis;
+ 3. frameworks e temas utilizados no desenvolvimento podem contribuir com páginas inacessíveis.
+
+Baseando-se nos [polyfills](https://remysharp.com/2010/10/08/what-is-a-polyfill) que permitem prover um recurso não suportado pelo browser foi desenvolvido o [HaTeMiLe](https://github.com/carlsonsantana/HaTeMiLe-for-JavaScript) (anagrama para HTML Accessible, removendo as consoantes de Accessible), uma biblioteca que utiliza os padrões web para tentar resolver alguns problemas de acessibilidade.
```

Comment:
Eu colocaria uma vírgula logo depois da palavra **polyfills**, e outra vírgula depois da palavra **browser**. Ficando assim:
`Baseando-se nos [polyfills](https://remysharp.com/2010/10/08/what-is-a-polyfill), que permitem prover um recurso não suportado pelo browser, foi desenvolvido...`